src/PerfView/StackViewer/CallTreeView.cs

```diff
@@ -481,16 +495,17 @@ public string InclusiveMetricByScenarioString
         /// </summary>
         /// <param name="maxDepth">Maximum depth to expand</param>
         /// <param name="expandGraphNodes">If true graph nodes (which are not guarnteed to terminate) are expanded. </param>
-        public void ExpandToDepth(int maxDepth, bool expandGraphNodes = false)
+        public void ExpandToDepth(int maxDepth, bool expandGraphNodes = false, bool selectExpandedNode = true)
         {
             if (maxDepth == 0)
                 return;
             if (!expandGraphNodes && Data.IsGraphNode)
                 return;
 
-            IsExpanded = true;
+            ExpandNode(selectExpandedNode);
+
             foreach (var child in VisibleChildren)
-                child.ExpandToDepth(maxDepth - 1, expandGraphNodes);
+                child.ExpandToDepth(maxDepth - 1, expandGraphNodes, selectExpandedNode);
```

Comment:
I note that previously the last node was selected, but now no node is selected.    I guess I am OK with no nodes being selected on expand-all so this is OK (nothing to do).